content/docs/1.7.0/deploy/important-notes/index.md

```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
+Online replica rebuilding was introduced in Longhorn 1.7.0, so offline replica rebuilding has been removed.
 
-### Support for ARM64 Platform
+### Block-type Disk Supports SPDK AIO, NVMe and VirtIO Bdev Drivers 
 
-As of Longhorn v1.6.0, volumes using the V2 Data Engine support the ARM64 platform. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+Before Longhorn v1.7.0, Longhorn block-type disks only supported the SPDK AIO bdev driver, which introduced extra performance penalties. Since v1.7.0, block devices can be directly managed by SPDK NVMe or VirtIO bdev drivers, improving IO performance through a kernel bypass scheme. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7672).
 
-### Replacing `backendStoreDriver` with `dataEngine`
+### Introduction of Filesystem Trim
 
-The attribute `backendStoreDriver`, which is defined in the parameters of StorageClasses and other Longhorn resources (for example, volumes, engines, and replicas), has been replaced with `dataEngine`. You must remove the existing StorageClasses for V2 volumes and create new ones that use `dataEngine`.
+Filesystem trim is supported since Longhorn v1.7.0. If a disk is managed by the SPDK AIO bdev driver, the Trim (UNMAP) operation is not recommended in a production environment (ref). It is recommended to manage a block-type disk with an NVMe bdev driver.
 
-### Updating the Linux Kernel on Longhorn Nodes
+### Linux Kernel on Longhorn Nodes
 
-Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. To prevent this, update the Linux kernel on Longhorn nodes to version 5.19 or later. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/). Additionally, version 6.7 or later is highly recommended for improved system stability.
 
-### Storage Network Support for Read-Write-Many (RWX) Volumes
+### Snapshot Creation Times Change from Time to Time
 
-Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, the network's reliance on Multus results in a significant restriction.
+Snapshots created before Longhorn v1.7.0 changes from Time to Time. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot info, leading to potential 1-second gaps between snapshots among different replicas. This bug was fixed in v1.7.0, but snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
```

Comment:
```suggestion
Snapshots created before Longhorn v1.7.0 have their Creation Time, as shown in the UI, change occasionally. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot info, leading to potential 1-second gaps between snapshots among different replicas. This bug was fixed in v1.7.0, but snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
```